libs/application/templates/driving-license/src/lib/drivingLicenseTemplate.ts

```diff
@@ -56,15 +56,15 @@ const template: ApplicationTemplate<
   Events
 > = {
   type: ApplicationTypes.DRIVING_LICENSE,
-  name: 'Umsókn um ökuskilríki',
+  name: m.applicationForDrivingLicense,
   dataSchema,
   readyForProduction: true,
   stateMachineConfig: {
     initial: States.DRAFT,
     states: {
       [States.DRAFT]: {
         meta: {
-          name: 'Umsókn um ökuskilríki',
+          name: 'Umsókn um ökuskírteini',
```

Comment:
Why not move all strings to messages.ts file?

Reply:
this is actually a string type, but I could refer to the title in the messages, will do,